animation_nodes/math/vector.pyx

```diff
@@ -118,10 +118,16 @@ cdef float distanceSquaredVec3(Vector3* a, Vector3* b):
 cdef float dotVec3(Vector3* a, Vector3* b):
     return a.x * b.x + a.y * b.y + a.z * b.z
 
+@cython.cdivision(True)
 cdef float angleVec3(Vector3 *a, Vector3 *b):
     cdef float dot = dotVec3(a, b)
-    cdef float val
-    val = dot / (lengthVec3(a) * lengthVec3(b))
+    cdef float val, lengthVecA, lengthVecB
+    lengthVecA = lengthVec3(a)
+    lengthVecB = lengthVec3(b)
```

Comment:
I think you should just assign `lengthVec3(a) * lengthVec3(b)` to a single variable, then check for zero, returning zero directly.

Reply:
Okay.

Reply:
Changed.